README.md

```diff
@@ -4,9 +4,9 @@ Welcome to di "Correct Pidgin Ipsum" site! For here, we gather beta pidgin ipsum
 
 ## 🌟 First of all Introduction
 
-What is Lorem Ipsum? Source `https://www.lipsum.com`
+What is Pidgin Ipsum? Source `https://www.lipsum.com`
 
-Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap, into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum, passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum.
```

Comment:
This was intentional, as Pidgin Ipsum is the derivative of Lorem ipsum.
I think it would be best to make the introduction, something like the Origins of Pidgin Ipsum, 

- Then we describe Lorem ipsum
- And then add the inspiration part of pidgin ipsum